docs/source/contributing.rst

```diff
@@ -167,6 +167,14 @@ to get feedback on, feel free to submit it as a PR. (In this case it's
 traditional to start the PR title with ``[WIP]``, for "work in
 progress".)
 
+When you are submitting your PR, you can include ``Closes #123``,
+``Fixes: #123`` or
+`some variation <https://help.github.com/en/articles/closing-issues-using-keywords>`__
+in either your commit message or the PR description, in order to
+automatically close the issue you are solving when the PR is merged.
+This is not mandatory, but it helps cleaning up and prevents dangling issues
```

Comment:
You already said "can", so I think the non-mandatoriness is apparent, and it might be crisper to do just ~"This keeps us closer to the desired state where each open issue reflects some work that still needs to be done."